### PR TITLE
SPCTR-3294: Fix - Testimonial - Resolved gradient styles issues.

### PR DESCRIPTION
**Pull request by rajmohanreddy774**
### Description

Gradient styles doesn't work as intended in Testimonial block.
https://brainstormforce.atlassian.net/browse/SPCTR-3294

### Screenshots

https://d.pr/v/YJVl23

### Checklist:
- [x] My code is tested
- [ ] My code follows accessibility standards 
- [ ] My code has proper inline documentation 
- [ ] I've included any necessary tests 
- [ ] I've included developer documentation 
- [x] I've added proper labels to this pull request

### CODE REVIEW

**Review by AbhijitNage123 — APPROVED** [review]

File: includes/blocks/testimonial/attributes.php [hunk 1]
```diff
@@ -135,8 +135,8 @@
 		'backgroundType'             => 'none',
 		'backgroundColor'            => '#eeeeee',
 		'backgroundHoverColor'       => '',
-		'gradientColor1'             => '',
-		'gradientColor2'             => '',
+		'gradientColor1'             => '#06558a',
+		'gradientColor2'             => '#0063A1',
```
Comment (bsf-zanev) [inline includes/blocks/testimonial/attributes.php:139]: Let's use a small A here and in JS.

File: README.md [hunk 2]
```diff
@@ -215,6 +215,9 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 
 ## Changelog ##
 
+### x.x.x ###
+* Fix: Testimonial - Resolved an issue wherein the gradient styles would not work as intended.
```
Comment (bsf-zanev) [inline README.md:219]:
```suggestion
* Fix: Testimonial - Resolved an issue wherein the gradient background styles would not work as intended.
```

File: readme.txt [hunk 3]
```diff
@@ -215,6 +215,9 @@ Our external packages use [Rating Star Component](https://github.com/n49/react-s
 
 == Changelog ==
 
+= x.x.x =
+* Fix: Testimonial - Resolved an issue wherein the gradient styles would not work as intended.
```
Comment (bsf-zanev) [inline readme.txt:219]:
```suggestion
* Fix: Testimonial - Resolved an issue wherein the gradient background  styles would not work as intended.
```

**Review by bsf-zanev — APPROVED** [review]
Looks good, jsut a minor changelog update and hex value update.

File: readme.txt (hunk 3 above)
Comment (rajmohanreddy774) [reply readme.txt]: updated.

File: README.md (hunk 2 above)
Comment (rajmohanreddy774) [reply README.md]: Done

**Review by adi3890 — APPROVED** [review]

**Pull request merged**